### PR TITLE
Modernise and adjust German translation

### PR DESCRIPTION
**Pull request by theFeu**
Referring to my [previous PR](https://github.com/confcodeofconduct/confcodeofconduct.com/pull/153) and the review given by dnsmichi.

### CODE REVIEW

**Comment by MattiSG** [comment]
@dnsmichi @mbrochh feel like reviewing this two-words PR? 😃

**Comment by mbrochh** [comment]
LGTM

File: index-de.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h2>Die Kurzfassung</h2>
       tolerieren keinerlei Belästigung von Konferenzbesucher_innen.
       Sexualisierte Sprache und Darstellungen sind auf der Konferenz
       unerwünscht. Dies umfasst den Veranstaltungsort, die Vorträge, Workshops
-      und Feten, sowie Twitter und andere Medien. Teilnehmer_innen, die gegen
+      und Feiern, sowie Twitter und andere Medien. Teilnehmer_innen, die gegen
```
Comment (dnsmichi) [inline index-de.html]: `Feiern` is a more common German word, also known in Austria.

File: index-de.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h2>Die längere Fassung</h2>
       Solltest du dich belästigt fühlen oder mitbekommen, dass jemand anders
       belästigt wird oder sonstige Probleme auftreten, wende dich bitte sofort
       an ein Mitglied des Veranstaltungspersonals. Diese sind an ihren
-      entsprechend markierten T-Shirts schnell zu erkennen.
+      entsprechend markierten T-Shirts und/oder Ausweisen schnell zu erkennen.
```
Comment (dnsmichi) [inline index-de.html]: This adds the badges as marker, which is a good thing. Not every organizer uses special t-shirts.

Comment (theFeu) [reply index-de.html]: The badges are also mentioned in the original, but were missing in the German translation!

Comment (dnsmichi) [reply index-de.html]: Ah, I see. Need more coffee.

**Comment by MattiSG** [comment]
Sie sind alle super, danke für die Zeile Übersetzung @dnsmichi!

**Pull request merged**